palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/config/ApplicationConfiguration.java

```diff
@@ -141,6 +142,11 @@ public PolicyService policyService(final PolicyClient policyClient, final Client
         return new PolicyService(policyClient, policyUriSupplier, getAsyncExecutor());
     }
 
+    @Bean
+    public PalisadeHealthIndicator dataHealthIndicator(final AuditService auditService, final PolicyService policyService, final ResourceService resourceService, final UserService userService) {
```

Comment:
dataHealthIndicator? Meant to be palisadeHealthIndicator?

Reply:
renamed